Mesh_3/doc/Mesh_3/CGAL/Mesh_cell_base_3.h

```diff
@@ -8,7 +8,7 @@ namespace CGAL {
 Compact_mesh_cell_base_3, because the later has a different API.
 -- Laurent Rineau, 2013/10/16
 \deprecated This class is deprecated since \cgal 4.3. Use
-`CGAL::Compact_mesh_cell_base_3<Gt,MD,Tds>` instead.
+`CGAL::Compact_mesh_cell_base_3<Gt,MD,CurveIndex,Tds>` instead.
```

Comment:
Why add a new parameter, that is identical to `typename MD::Curve_index`?

Reply:
There is no such type in the Concept MeshDomain_3, in the doc of Polyhedral_mesh_domain_3, and apparently `there is no type Curve_index in class CGAL::Polyhedral_mesh_domain_3`

Reply:
That is right. I understand, now.